src/main/java/org/ow2/proactive/catalog/graphql/bean/argument/CatalogObjectRevisionWhereArgs.java

```diff
@@ -35,13 +36,13 @@
  */
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class RevisionWhereArgs extends BaseWhereArgs<Long> {
+public class CatalogObjectRevisionWhereArgs extends WhereArgs<Long> {
 
     private final Boolean isLatest;
 
-    public RevisionWhereArgs(Long eq, Long ne, Long gt, Long gte, Long lt, Long lte, Long[] in, Long[] notIn,
-            Boolean isNull, Boolean isLatest) {
-        super(eq, ne, gt, gte, lt, lte, in, notIn, isNull);
+    @Builder
```

Comment:
Could you please add a comment before the constructor of WhereArgs.java to explain what does this eq, ne, gt ... etc.. mean